packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -0,0 +1,64 @@
+import { attr } from '@microsoft/fast-element';
+import { DesignSystem, FoundationElement } from '@microsoft/fast-foundation';
+import { template } from '../theme-provider/template';
+import { styles } from './styles';
+import {
+    WaferMapColorBy,
+    WaferMapColorsScale,
+    WaferMapDataType,
+    WaferMapDie,
+    WaferMapOrientation,
+    WaferMapQuadrant
+} from './types';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-wafer-map': WaferMap;
+    }
+}
+
+/**
+ * A nimble-styled WaferMap
+ */
+export class WaferMap extends FoundationElement {
+    @attr({
+        attribute: 'wafer-quadrant'
+    })
+    public waferQuadrant!: WaferMapQuadrant;
+
+    @attr({
+        attribute: 'wafer-orientation'
+    })
+    public waferOrientation!: WaferMapOrientation;
```

Comment:
I'm curious why the wafer prefix was added to some of these attributes.

I think it will look redundant in practice:
```html
<nimble-wafer-map wafer-orientation="top" wafer-quadrant="top-left"></nimble-wafer-map>
```

vs

```html
<nimble-wafer-map orientation="top" quadrant="top-left"></nimble-wafer-map>
```

Are there other attributes that may be named orientation or quadrant expected on the wafer-map? If that is the case and the preference is to keep the wafer prefix for some properties then the Type names should be updated to match the convention (i.e. `WaferMap<PropertyName>`):

```ts
public waferOrientation: WaferMapWaferOrientation;
public waferQuadrant: WaferMapWaferQuadrant;
```

If we do not expect other possible conflicts then I'd prefer the simpler name:

```ts
public orientation: WaferMapOrientation;
public quadrant: WaferMapQuadrant;
```

I might ask if we should go a step further, do we expect other wafer type controls besides this one (i.e. `wafer-<something-else>`)? If we do not expect other wafer components, can we simplify the component name to just `nimble-wafer`?

And then the types become:

```ts
public orientation: WaferOrientation;
public quadrant: WaferQuadrant;
```

and the html is a lot less redundant and verbose:
```html
<nimble-wafer orientation="top" quadrant="top-left"></nimble-wafer>
```

Reply:
I changed the name for the attribute to have without "wafer-".
We had a sync with our PO and the name of the component can remain "nimble-wafer-map" because in the future we will want to create also the "Heat map" (https://en.wikipedia.org/wiki/Heat_map#Types), so in this context it make sense to keep here also the "map" word. 
So, I updated and kept your suggestion for the attributes names.